packages/core/backend/sirius-components-collaborative/src/main/java/org/eclipse/sirius/components/collaborative/representations/api/IEditingContextEventProcessorSubscriptionSchedulerProvider.java

```diff
@@ -0,0 +1,28 @@
+/*******************************************************************************
+ * Copyright (c) 2025 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.sirius.components.collaborative.representations.api;
+
+import reactor.core.scheduler.Scheduler;
+
+/**
+ * Used to provide a different scheduler for subscription of an editing context event processor.
+ *
+ * <p>
+ * Thanks to this API, it will be easier to customize the thread management strategy of subscriptions in an editing context event processor.
+ * </p>
+ *
+ * @author gcoutable
+ */
+public interface IEditingContextEventProcessorSubscriptionSchedulerProvider {
```

Comment:
I'll just update the name since its not providing a `Scheduler` for `IEditingContextEventProcessor` but for `EventProcessorSubscriptionProvider` it works mostly for representation event processors, not the editing context event processor.